### PR TITLE
Fix closure errors with MINIMAL_RUNTIME + USE_PTHREADS. NFC.

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/preamble_minimal.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ if (!ENVIRONMENT_IS_PTHREAD) {
 #if USE_PTHREADS
 } else {
 #if MODULARIZE
-  updateGlobalBufferAndViews({{{EXPORT_NAME}}}.buffer);
+  updateGlobalBufferAndViews(Module.buffer);
```
Comment (kripken) [inline src/preamble_minimal.js:111]: Why is this changed?

Comment (sbc100) [reply src/preamble_minimal.js:111]: Without this it fails closure compiler with `MyModule` not defined.  I think this is because closure runs before modularization so at the point of closure compiler `MyModule` (the EXPORT_NAME) does not exist.

However `Module.buffer` works just fine here because the modularized code starts with

```
function(MyModule) {                                                             
  MyModule = MyModule || {};                                                     
                                                                                 
                                                                                 
                                                                                 
var Module = MyModule;  
```

Comment (kripken) [reply src/preamble_minimal.js:111]: I see, sounds good. And closure will remove the extra var anyhow.

**Review by kripken — APPROVED** [review]
